src/CWTConstruction.jl

```diff
@@ -58,6 +58,8 @@ end
     CWT(wave::ContWaveClass, Q=8, boundary::WaveletBoundary=SymBoundary(),
     averagingType::Average = Father(), averagingLength::Int = 4, frameBound=1, p::N=Inf, β=4)
 """
+
```

Comment:
this particular pair of spaces will actually wreck the documentation. The others are definitely improvements though.